_sources/CSPPracticeQuestions/exam24_3.rst

```diff
@@ -23,10 +23,10 @@ You will not be able to change your answers after you hit the "Finish Exam" butt
        :answer_c: enter.sample.org
        :answer_d: sample.org/help
        :correct: c
-       :feedback_a: Incorrect
-       :feedback_b: Incorrect
-       :feedback_c: Correct
-       :feedback_d: Incorrect
+       :feedback_a: Incorrect. This is a completely different website unrelated to sample.org.
+       :feedback_b: Incorrect. Sample.org is a subdomain of biz in this answer choice.
```

Comment:
I think sample.org should always be lowercase. How sure how annoying it is to fix this though! I can fix it after merging.